README.md

```diff
@@ -29,3 +29,6 @@
   * could still use some love, but isn't so bad anymore
 
 
+### Troubleshooting:
+
+* On OSX El Captain, gem installation could fail with a permission error. See: http://stackoverflow.com/questions/32659348/operation-not-permitted-when-on-root-el-capitan-rootless-disabled
```

Comment:
Agree... But how to prevent such issues then ? Because I am not allowed to install gems on my computer without sudo :-( Perhaps we should tell mac users to go with a different way ? In my case I just used the ruby installation that was there.